CHANGELOG.md

```diff
@@ -21,33 +21,68 @@ If possible, provide tooling that performs the changes, e.g. a shell-script.
 
 ## New features
 
+#### Alphabet
+
+* Added the seqan3::dna3bs alphabet that models bisulfite-treated dna4 sequence (#1191).
+
 #### Alignment
-* The score type used in the alignment score matrix and the result type is now configurable through a template
+
+* The function seqan3::align_pairwise can be parallelised using the seqan3::align_cfg::parallel configuration.
+* The score type used in the alignment score matrix and the result type is configurable through a template
   argument of the seqan3::align_cfg::result configuration.
 
 #### Argument parser
-* Simplified reading file extensions from formatted files in the input/output file validators.
+
+* Simplified reading file extensions from formatted files with the seqan3::input_file_validator and
+  seqan3::output_file_validator (#863).
 * The seqan3::value_list_validator is now constructible from a range or a parameter pack.
 * Enable subcommand argument parsing ([How-to](https://docs.seqan.de/seqan/3-master-user/subcommand_arg_parse.html)).
+* The seqan3::argument_parser::add_option (and add_positional_option) calls allow enum types when using the
+  seqan3::enumeration_names customisation point (#1196).
 
 #### Build system
 
 * `find_package(SeqAn3)` is now case-insensitive and always populates `SEQAN3_*` variables in all upper-case.
 
 #### Core
+
+* Added seqan3::lzcnt, seqan3::tzcnt, and seqan3::popcount for bit manipulation (#1141).
 * Added traits for "metaprogramming" with `seqan3::type_list` and type packs.
+* Added the SIMD functions seqan3::upcast and seqan3::upcast_signed (#1190).
 
 #### I/O
 
+* Support of padded alignments in the SAM/BAM format (#1173) was added.
+* We increased our input performance using a faster iterator on the stream buffer.
 * Asynchronous input (background file reading) supported via seqan3::view::async_input_buffer.
 * Reading field::cigar into a vector over seqan3::cigar is supported via seqan3::alignment_file_input.
 * Writing field::cigar into a vector over seqan3::cigar is supported via seqan3::alignment_file_output.
 
+### Range
+
+* Added the seqan3::dynamic_bitset (#1153).
+* Added the seqan3::views::translate_join (#1171).
+* Added the seqan3::views::to_simd (#1190).
+* Added the seqan3::views::kmer_hash (#946).
+
+#### Search
+
+* The memory footprint of FM-indices over text collections was reduced (#1363).
+
+### Std
+
+* We provide a `std::to_chars` overload for floating point data types in our `seqan3/std/from_chars` header (#1160).
+
 ## API changes
 
 * **Customising for third party types has changes slightly:**
   You are only affected if you added types to `seqan3::custom::`.
   Please see [About Customisation](http://docs.seqan.de/seqan/3-master-user/about_customisation.html).
+* All our concepts are named in the `snake_case` style (e.g. `seqan3::WritableAlphabet` -> `seqan3::writable_alphabet`)!
+
+#### Alphabet
+
+* The seqan3::cigar alphabet is not an seqan3::alphabet anymore but only a seqan3::semialphabet (#1285).
```

Comment:
haven't we split them into writable and not writable concepts as well?

Reply:
This was done in #979 in May 2019 but we release 3.0.0 in June 2019 so it was already in the code base.